utils/src/wizard/AzureWizard.ts

```diff
@@ -188,17 +189,19 @@ export class AzureWizard<T extends (IInternalActionContext & Partial<types.Execu
 
             let step: AzureWizardExecuteStep<T> | undefined = steps.pop();
             while (step) {
+                const start: Date = new Date();
```

Comment:
nit: Small detail, but this should probably go on line 205-206 where the step is executed as part of the try/finally clause,

Reply:
Although I agree and initially did it this way too, I think if we omit the other step methods being run, the individual step times will not sum up to the total recorded time as closely, and that could look weird to our users if they are paying close attention (more noticeable on longer commands).  For that reason, I thought it might be better to include the other methods in the calculation as well, but am happy to discuss if we want to only measure the execute method.